scipy/signal/_support_alternative_backends.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+from scipy._lib._array_api import support_alternative_backends, SCIPY_ARRAY_API
+from . import _signaltools
+# These don't really need to be imported, but otherwise IDEs might not realize
+# that these are defined in this file / report an error in __init__.py
+from ._signaltools import (
+    convolve, correlate, fftconvolve,  # noqa: F401
+)
+
+_generic_implementations = {}
+
+array_special_func_map = {
```

Comment:
```suggestion
array_signal_func_map = {
```

Reply:
Probably will just make it `array_func_map` or so. Or TBH might remove this and use the decorator for everything? The reason we didn't do that in `special` is that there's not a Python function to decorate, but for the other modules, it would probably be nicer?